mem-05-H.xml

```diff
@@ -4388,7 +4388,7 @@ Watch Gowron say this: {YouTube video:url:http://youtu.be/KArQlviVXMk}</column>
       <column name="entry_name">HIr</column>
       <column name="part_of_speech">v:is</column>
       <column name="definition">be stuck (physically fastened to something)</column>
-      <column name="definition_de">abgestempelt</column>
+      <column name="definition_de">abgestempelt sein</column>
```

Comment:
festsitzen, festkleben
[as a side note: "be stuck" is not an adjective in German, in neither form]